mavros_extras/CMakeLists.txt

```diff
@@ -111,6 +111,7 @@ add_library(mavros_extras_plugins SHARED
   src/plugins/obstacle_distance.cpp
   src/plugins/odom.cpp
   src/plugins/onboard_computer_status.cpp
+  src/plugins/open_drone_id.cpp
```

Comment:
This part of file must be generated by cog script. Please do not change this blocks manually.